redis/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+
+services:
```

Comment:
Please include this in docker-compose.fvm.yml instead

Reply:
ok kub